### ISSUE TITLE
Add ignore_time option to integration tests

### ISSUE THREAD

**Issue by okynos** [wazuh/wazuh-qa#665]
Hello team,

We want to to test Vulnerability detector specific case, this issue is related to test the `ignore_time` option.
This test is related to https://github.com/wazuh/wazuh-qa/issues/660

We have to test the following points:
- Debian manager.
- RPM manager.

Regards.

**Comment by DFolchA** [comment]
Hello team, 
I am working on creating functions to generate vulnerability detector alerts, by adding entries to the corresponding databases.
Regards,
Daniel Folch

**Comment by DFolchA** [comment]
Hello team,
I have finished the functions to mock vulnerability detector alerts, there are two functions:

- **insert_package_in_db**: Inserts a new package in the database of installed packages.
- **insert_vulnerability**: Inserts a new vulnerability entry in the database of vulnerabilities.

When used together it is possible to generate custom alerts like this one:

```
** Alert 1588947655.225730: mail  - vulnerability-detector,gdpr_IV_35.7.d,pci_dss_11.2.1,pci_dss_11.2.3,
2020 May 08 14:20:55 centos7->vulnerability-detector
Rule: 23506 (level 13) -> 'Wazuh integration test vulnerability'
{"vulnerability":{"cve":"WVE-000","title":"Wazuh integration test vulnerability","severity":"Critical","published":"2020-05-08T14:04:25Z","updated":"","state":"Fixed","cvss":{"cvss2":{"vector":{"attack_vector":"network","access_complexity":"low","authentication":"none","confidentiality_impact":"low","integrity_impact":"complete","availability":"complete"},"base_score":10},"cvss3":{}},"package":{"name":"WazuhIntegrationPackage","version":"1.0.0","architecture":"x86_64","condition":"Package less than 2.0.0-1.el7"},"advisories":"RHSA-2010:0029","cwe_reference":"WVE-000 -> WVE-001","bugzilla_reference":"https://github.com/wazuh/wazuh-qa","reference":"https://github.com/wazuh/wazuh-qa"}}
vulnerability.cve: WVE-000
vulnerability.title: Wazuh integration test vulnerability
vulnerability.severity: Critical
vulnerability.published: 2020-05-08T14:04:25Z
vulnerability.state: Fixed
vulnerability.cvss.cvss2.vector.attack_vector: network
vulnerability.cvss.cvss2.vector.access_complexity: low
vulnerability.cvss.cvss2.vector.authentication: none
vulnerability.cvss.cvss2.vector.confidentiality_impact: low
vulnerability.cvss.cvss2.vector.integrity_impact: complete
vulnerability.cvss.cvss2.vector.availability: complete
vulnerability.cvss.cvss2.base_score: 10
vulnerability.package.name: WazuhIntegrationPackage
vulnerability.package.version: 1.0.0
vulnerability.package.architecture: x86_64
vulnerability.package.condition: Package less than 2.0.0-1.el7
vulnerability.advisories: RHSA-2010:0029
vulnerability.cwe_reference: WVE-000 -> WVE-001
vulnerability.bugzilla_reference: https://github.com/wazuh/wazuh-qa
vulnerability.reference: https://github.com/wazuh/wazuh-qa
```

I have also added a function called:

- **update_package_info**: modifies an existing entry od the packages database, this way we can mock upgrades and downgrades.

Regards,
Daniel Folch

**Comment by BraulioV** [comment]
Closed by #721
